### PR TITLE
Add widget that display joined array entries

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Add widget that display joined array entries

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/Extensibility/components/Widget.js [hunk 1]
```diff
@@ -40,10 +41,12 @@ export function Widget({ structure, value, inlineRenderer, ...props }) {
       <Renderer {...props} />
     );
 
-  return Array.isArray(childValue) && !Renderer.array ? (
-    childValue.map(item => (
-      <SingleWidget value={item} structure={structure} {...props} />
-    ))
+  return Array.isArray(childValue) &&
```
Comment (qbalukom) [inline core-ui/src/components/Extensibility/components/Widget.js:44]: Shouldn't this be solved by simply assigning `JoinedArray.array = true`?

**Review by qbalukom — CHANGES_REQUESTED** [review]

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -1009,10 +1009,13 @@ extensibility:
       docs-info: For more information, see <0>Extensibility documentation</0>.
       error: Failed to upsert UI configuration.
       no-columns: No example values could be inferred from the CRD.
+      not-an-array: Wrong data format. Values sgould be an array.
```
Comment (majakurcius) [inline core/src/i18n/en.yaml:1012]:
```suggestion
      not-an-array: Wrong data format. Values should be an array.
```

File: docs/extensibility/display-widgets.md [hunk 3]
```diff
@@ -43,6 +43,27 @@ The following values are automatically handled:
 
 <img src="./assets/display-widgets/Badge.png" alt="Example of a badge widget" width="20%" style="border: 1px solid #D2D5D9">
 
+### JoinedArray
+
+JoinedArray widgets render all the values of an array of strings as a comma-separated list.
+
+#### Widget-specific parameters
+
+- **separator** - a string by which the elements of the array will be separated by. The default value is a comma `,`
```
Comment (majakurcius) [inline docs/extensibility/display-widgets.md:52]:
```suggestion
- **separator** - a string by which the elements of the array will be separated by. The default value is a comma `,`.
```

**Review by majakurcius — APPROVED** [review]

**Comment by qbalukom** [comment]
I'd also suggest including a more meaningful example screenshot - with actual words instead of single letters.

**Review by majakurcius — APPROVED** [review]

**Review by qbalukom — DISMISSED** [review]

**Review by qbalukom — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Pull request merged**